.github/workflows/publish.yaml

```diff
@@ -0,0 +1,45 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: 'Publish Component Library'
+
+# Event for the workflow
+on:
+  pull_request_target:
+    branches:
+      - master
+    types:
+      - closed
+    paths:
+      - packages/components/**            
+  push:
+    branches-ignore:
+      - 'master'
+    paths:
+      - packages/components/**      
+
+# List of jobs
+jobs:
+  npm-publish:
+    # Operating System
+    runs-on: ubuntu-latest
+    # Job steps
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dependencies, build
+        run: yarn workspace @calyptia-vivo/components install && yarn workspace @calyptia-vivo/components build
+      - name: Configure git
+        run: git config --global user.name "Continuous Integration" && git config --global user.email "ci@calyptia.com"         
```

Comment:
Is this required? Checkout is above and configures it for the specific author and commiter.